go.mod

```diff
@@ -3,57 +3,54 @@ module github.com/testcontainers/testcontainers-go
 go 1.19
 
 require (
-	github.com/cenkalti/backoff/v4 v4.2.0
-	github.com/containerd/containerd v1.6.19
+	github.com/cenkalti/backoff/v4 v4.2.1
```

Comment:
I'd revert any change in the root module regarding dependencies, as we reserve that task to dependabot. Otherwise, all the existing modules importing could need to bump their dependencies here.

Could you please rollback the changes in the go.mod and go.sum here? It could probably lead to changes in the nats module